### DISCUSSION TITLE
Mixed Chart - Stacked Bar metrics sort

### DISCUSSION THREAD

**Discussion by marianysilva** [apache/superset#28392] [Ideas]
I would like the sorting of metrics in a Mixed Chart with Stacked Bars to impact the data visualization. I named my metrics as 1, 2, 3 in an attempt to sort them, but it was unsuccessful. I also changed the order in which the metrics were declared, but this did not reorder them.

![Screenshot 2024-05-08 at 18 56 38](https://github.com/apache/superset/assets/97555693/12dd59ac-b63c-4a56-9b3b-dd2a50b1f84b)
![Screenshot 2024-05-08 at 18 55 45](https://github.com/apache/superset/assets/97555693/34514db5-e934-4e58-bf74-a374be22bccb)

**Comment by nilmonto** [comment]
I've tried to do exactly the same thing and was abble to accomplish this by editing the metric labels in alphabetic way (numbers are not considered), so, if you try: { A - 1, B - 2, C - 3 } and after { A - 3, B - 2, C - 1 } you would be able to see the difference.

**Reply by edwardzhanged** [reply]
I don't think it is a good solution. The value may change regardless of alphabetic order.
